imap_processing/cdf/config/imap_hi_variable_attrs.yaml

```diff
@@ -312,6 +312,20 @@ hi_pset_esa_energy_step:
   VAR_TYPE: support_data
   dtype: uint8
 
+hi_pset_calibration_prod:
+  CATDESC: Calibration product number
+  FIELDNAM: Coincidence types are combined into various calibration products
```

Comment:
Should the CATDESC and FIELDNAM values be switched? I've been seeing longer descriptions being placed in the CATDESC field. FIELDNAM can be used to label a plot, so if that's the information that needs to be displayed, then switching these descriptions may not be necessary.

Reply:
Good catch!